src/SFA.DAS.EmployerApprenticeshipsService.Application/Commands/RemoveLegalEntity/RemoveLegalEntityCommandValidator.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using SFA.DAS.EAS.Application.Validation;
+using SFA.DAS.EAS.Domain.Data.Repositories;
+using SFA.DAS.EAS.Domain.Interfaces;
+using SFA.DAS.EAS.Domain.Models.EmployerAgreement;
+using SFA.DAS.EAS.Domain.Models.UserProfile;
+
+namespace SFA.DAS.EAS.Application.Commands.RemoveLegalEntity
+{
+    public class RemoveLegalEntityCommandValidator : IValidator<RemoveLegalEntityCommand>
```

Comment:
Maybe going forward we can do this it can be internal but we'd obviously have to change the instantiating of the object.